oscore-cf/pom.xml

```diff
@@ -0,0 +1,31 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.californium</groupId>
+    <artifactId>parent</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>oscore-cf</artifactId>
+  <dependencies>
+	<dependency>
+		<groupId>com.augustcellars.cose</groupId>
+		<artifactId>cose-java</artifactId>
+		<version>0.9.7</version>
+	</dependency>
+	<dependency>
+		<groupId>org.eclipse.californium</groupId>
+		<artifactId>scandium</artifactId>
+		<version>2.0.0-SNAPSHOT</version>
+		</dependency>
+  	<dependency>
+	 	<groupId>com.upokecenter</groupId>
+  		<artifactId>cbor</artifactId>
+  		<version>2.4.1</version>
+  	</dependency>
+  	<dependency>
+		<groupId>com.github.peteroupc</groupId>
+  		<artifactId>numbers</artifactId>
+  		<version>0.2.1</version>
```

Comment:
COSE Java 0.9.7 POM declares the following
```
<dependency>
  <groupId>com.github.peteroupc</groupId>
  <artifactId>numbers</artifactId>
  <version>0.3.0</version>
</dependency>
```
Any particular reason to use 0.2.1. instead?

Reply:
I do not think so. I will update.